### DISCUSSION TITLE
What are the requirements of Longhorn?

### DISCUSSION THREAD

**Discussion by Darin755** [longhorn/longhorn#9170] [Q&A] [answered]
I am building a cluster but I only have gigabit between the hosts which is shared with Proxmox and the actual services. How much bandwidth does longhorn pull? I am looking to have a handful of services and there may be a significant number of small writes. How much of a risk is there that longhorn will eat a significant amount of bandwidth? My storage is all ZFS sata so from a IO perspective the performance isn't exactly out of the water.

**Comment by derekbit** [comment]
Hello @Darin755 
Thanks for asking. Can you share more information about your hardware setup including CPU, RAM and Storage (SSD or HDD)?

**Reply by Darin755** [reply]
Is it heavily CPU bound? All my machines have mid tier CPUs and I was planning on only giving the VM a few cores. (For context one machine has a desktop i5 8th)

As far as ram goes I can give each VM up to 8gb of ram. In reality each VM will probably have 6gb as ZFS on the hypervisor takes a decent amount of ram. Does longhorn eat a lot of ram like Ceph does?

**Reply by derekbit** [reply, marked as answer]
> Is it heavily CPU bound?

It depends on how many volumes and if the workloads are IO intensive. `i5 8th` shall be okay for Longhorn.

> Does longhorn eat a lot of ram like Ceph does?

No, Longhorn doesn't consume too much ram.


Are the disks on host SSD or HDD?

The only concern is the 1 Gbps NIC in your cluster, but I think you can give Longhorn a try.

**Reply by Darin755** [reply]
Thanks
